semantics.json

```diff
@@ -223,6 +223,39 @@
                   "label": "Seek to after loops amounts",
                   "description": "Enter timecode in the format M:SS"
                 },
+                {
+                  "name": "wrongAnswerParams",
+                  "label": "Wrong Answer Params",
```

Comment:
I would use "Incorrect" instead of "Wrong"
Basically the same meaning but incorrect doesn't sound so harsh.

Reply:
fixed